api/backend/ecs/task_manager.go

```diff
@@ -171,13 +152,9 @@ func modelFromTasks(tasks []*ecs.Task) (*models.Task, error) {
 	}
 
 	model := &models.Task{
-		EnvironmentID: id.ClusterARNToECSEnvironmentID(*tasks[0].ClusterArn).L0EnvironmentID(),
-		PendingCount:  pendingCount,
-		RunningCount:  runningCount,
-		DesiredCount:  int64(len(tasks)),
-		TaskID:        id.ECSTaskID(*tasks[0].StartedBy).L0TaskID(),
-		Copies:        copies,
-		DeployID:      id.TaskDefinitionARNToECSDeployID(*tasks[0].TaskDefinitionArn).L0DeployID(),
+		RunningCount: runningCount,
```

Comment:
Why don't we want `EnvironmentID` in this model?

Reply:
that is handled in the logic layer